packages/headless/src/features/redirection/redirection-analytics-actions.ts

```diff
@@ -9,6 +9,7 @@ export const logTriggerRedirect = makeAnalyticsAction(
   (client, state) => {
     if (state.redirection && state.redirection.redirectTo !== null) {
       return client.logTriggerRedirect({
+        //shouldn't this be state.triggers.redirectTo? make another action?
         redirectedTo: state.redirection.redirectTo,
```

Comment:
In the controller,  we refer to `state.triggers.redirectTo`. I'm wondering whether the engine `redirection` state is still useful since it is now being handled by the `trigger` state.

Reply:
My understanding is that the engine `redirection` slice is used by the standalone search box.

However, I'm not sure why this is the analytics action used by the standalone searchbox...

Personally, unless we're worried it would be too breaking, I would be okay with renaming this action (not `client.logTriggerRedirect`) to something more generic and introducing a more explicit action for triggers.

Reply:
That seems like a good solution to me.